### ISSUE TITLE
Jump to PDF page when using gg with pdf.js

### ISSUE THREAD

**Issue by abbradar** [The-Compiler/qutebrowser#2020]
This is another feature request similar to https://github.com/The-Compiler/qutebrowser/issues/1916. We have `Ngg` hotkey, which, like in Vim, jumps to the Nth line. Line, however, is a poorly defined object in Web ~_^. However, there are places where we can actually do something more meaningful. One of those places is PDF.js where we can use this hotkey to jump to a page.

As was discussed in #2001, it's not always desirable to have this kind of magic in qutebrowser, so perhaps we need to add another parameter to the go-to command (`:smart`, like @Kingdread suggested?). FWIW, currently `Ngg` doe nothing on a PDF.js page.

**Comment by The-Compiler** [comment]
Note that that's a percentage value on normal websites, not lines :wink:

I'll let @Kingdread comment on the rest.

**Comment by The-Compiler** [comment]
Note that that's a percentage value on normal websites, not lines :wink:

I'll let @Kingdread comment on the rest.

**Comment by abbradar** [comment]
Oh, I'v been looking on a site with practically no vertical scrolling when I experimented with `gg`! Silly me, thanks.

**Comment by Kingdread** [comment]
Yeah, pdf.js renders the content into a scrollable div, which semms to be not ignored when using `gg`. Scrolling to a percentage value doesn't seem to be possible with the pdf.js API (currently), but scrolling to a page is definitely possible! So we could make `Ngg` scroll to the Nth page, and `G` scroll to the last page.

However, I first want to take care of #2001 first before starting this, because it contains some of the required logic ("is this page a pdfjs page").
